modules/messages/src/lib.rs

```diff
@@ -708,6 +720,9 @@ fn send_message<T: Config<I>, I: 'static>(
 		.send_message(encoded_payload)
 		.map_err(Error::<T, I>::MessageRejectedByPallet)?;
 
+	// return number of messages in the queue to let sender know about its state
+	let enqueued_messages = lane.data().queued_messages().checked_len().unwrap_or(0);
```

Comment:
@svyatonik 
I see two usages of `.queued_messages().checked_len().unwrap_or(0);`,
would it be worth to wrap it to `OutboundLaneData` e.g.
```
	pub fn queued_messages_len(&self) -> MessageNonce {
		self.queued_messages().checked_len().unwrap_or(0)
	}
``

Reply:
True. Let's add another method to `RangeInclusiveExt` - i.e. `saturated_len()`. I think it is used everywhere, except one single place where we actually need the `checked_` version. I'll do that a bit later

Reply:
done